tests/ui_tests/conftest.py

```diff
@@ -172,12 +171,14 @@ def _write_json_file(f_name, j_data):
 def login_to_adcm_over_api(app_fs, adcm_credentials):
     """Perform login via API call"""
     login_endpoint = f'{app_fs.adcm.url.rstrip("/")}/api/v1/token/'
-    app_fs.driver.get(app_fs.adcm.url)
+    LoginPage(app_fs.driver, app_fs.adcm.url).open().wait_page_is_opened()
```

Comment:
You don't need to explicitly wait for the page to be opened, it's implicit in open(). `wait_page_is_opened` exists for cases, when we open page by click on some UI element, for example

Reply:
fixed